tests/smoke/test_newsrec_model.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+import os
+import papermill as pm
+from reco_utils.recommender.newsrec.newsrec_utils import prepare_hparams
+from reco_utils.recommender.deeprec.deeprec_utils import download_deeprec_resources  
+from reco_utils.recommender.newsrec.models.base_model import BaseModel
+from reco_utils.recommender.newsrec.models.nrms import NRMSModel
+from reco_utils.recommender.newsrec.models.naml import NAMLModel
+from reco_utils.recommender.newsrec.models.lstur import LSTURModel
+from reco_utils.recommender.newsrec.models.npa import NPAModel
+from reco_utils.recommender.newsrec.IO.news_iterator import NewsTrainIterator, NewsTestIterator
+from reco_utils.recommender.newsrec.IO.naml_iterator import NAMLTrainIterator, NAMLTestIterator
+
+
+@pytest.fixture
```

Comment:
it would be better to use a temporary directory so that test resources do not accumulate on any build machines. https://docs.pytest.org/en/latest/tmpdir.html

Reply:
good suggestion, we have a fixture that generates a tmp file that is removed when the tests are finished: https://github.com/microsoft/recommenders/blob/master/tests/conftest.py#L23, just need to add `tmp` in the function for it to work, like here https://github.com/microsoft/recommenders/blob/master/tests/integration/test_movielens.py#L69 (there is no need to import because it is a fixture)